### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by K3nt3L** [lllyasviel/Fooocus#1033] [Show and tell]
I have just finished the translation into French (two versions : one for the entire interface, including styles, and the other without styles). In both cases, I did not translate the 'debug' section (guess french devs don't need translation).
[fr.json](https://github.com/lllyasviel/Fooocus/files/13463804/fr.json)
[fr_no_style.json](https://github.com/lllyasviel/Fooocus/files/13463805/fr_no_style.json)

I had to make some fairly drastic choices for words that do not exist in French. For example, instead of 'pyrocanny,' I used 'détection de contours' (contour detection). As for the styles, I assumed that users were not familiar with English at all, except for some styles that are identical in both languages (underground for example).

[https://github.com/K3nt3L/Foocus-french-translation](https://github.com/K3nt3L/Foocus-french-translation)

**Comment by Hexawar** [comment]
Merci

**Reply by K3nt3L** [reply]
Pas de quoi :) Have fun.

**Comment by triclic** [comment]
Bonjour, 
comment fait on pour changer la langue dans Fooocus. j'ai bien récupérer le fichier que j'ai placé dans le répertoire language, mais après ???
merci d'avance de votre aide

**Reply by Didier3L** [reply]
> Bonjour, comment fait on pour changer la langue dans Fooocus. j'ai bien récupérer le fichier que j'ai placé dans le répertoire language, mais après ??? merci d'avance de votre aide

python entry_with_update.py --language fr

**Reply by triclic** [reply]
Merci
